src/system.jl

```diff
@@ -387,11 +460,68 @@ Base.@kwdef mutable struct SystemDA <: System
 
     # Virtuals/PSD time series
     "Increment bids time series data. `KeyedArray` where the axis keys are `bid ids x datetimes`"
-    increments::KeyedArray{Vector{Tuple{Float64, Float64}}, 2}
+    increments::KeyedArray{PriceSensitiveBid, 2}
     "Decrement bids time series data. `KeyedArray` where the axis keys are `bid ids x datetimes`"
-    decrements::KeyedArray{Vector{Tuple{Float64, Float64}}, 2}
+    decrements::KeyedArray{PriceSensitiveBid, 2}
     "Price sensitive load bids time series data. `KeyedArray` where the axis keys are `bid ids x datetimes`"
-    price_sensitive_loads::KeyedArray{Vector{Tuple{Float64, Float64}}, 2}
+    price_sensitive_loads::KeyedArray{PriceSensitiveBid, 2}
+end
+
+"""
+    $TYPEDEF
+
+Subtype of a `System` for modelling a purely financial style of day-ahead market clearing.
+Key differences to the physical day-ahead market clearing approach are that _all_ offers and
+bids are price sensitive and virtual participants are not distinguished from physical
+participants.
+
+Fields:
+$TYPEDFIELDS
+"""
+Base.@kwdef mutable struct FinancialSystemDA <: System
+    "`Dictionary` where the keys are bus names and the values are generator ids at that bus"
+    gens_per_bus::Dictionary{BusName, Vector{String}}
+    "`Dictionary` where the keys are bus names and the values are increment bid ids at that bus"
+    energy_offers_per_bus::Dictionary{BusName, Vector{BidName}}
```

Comment:
Are these similar to `incs_per_bus` in the other systems? Should we try to keep the names consistent?

Reply:
Yes, I think they are identical in terms of type.  It's a question of different concepts.  `incs`/`increments` refer specifically to virtual bids, whereas here the `energy_offers` can be physical or virtual and we don't have a way to distinguish between the two.